resources/keybinds.txt

```diff
@@ -29,20 +29,27 @@ KEYBINDINGS:
 [TOGGLE_STOCKS:KEY_I]
 [TOGGLE_ZONES:KEY_U]
 [TOGGLE_OCCLUSION:KEY_O]
+[TOGGLE_FOG:KEYS_TILDE]
 [TOGGLE_CREATURE_MOODS:KEY_M]
 [TOGGLE_CREATURE_PROFS:KEY_P]
 [TOGGLE_CREATURE_JOBS:KEY_J]
 [TOGGLE_CREATURE_NAMES:KEY_N]
 [CHOP_WALLS:KEY_C]
 [CYCLE_TRACKING_MODE:KEY_F]
 [RESET_VIEW_OFFSET:KEY_Z]
-[DECR_SEGMENT_Z:KEY_1]
-[INCR_SEGMENT_Z:KEY_2]
+[DECR_SEGMENT_Z:KEYPAD_3]
+[INCR_SEGMENT_Z:KEYPAD_9]
+[DECR_SEGMENT_X:KEYPAD_4]
+[INCR_SEGMENT_X:KEYPAD_6]
+[DECR_SEGMENT_Y:KEYPAD_2]
+[INCR_SEGMENT_Y:KEYPAD_8]
 [TOGGLE_SINGLE_LAYER:KEY_S]
 [TOGGLE_SHADE_HIDDEN_TILES:KEY_B]
 [TOGGLE_SHOW_HIDDEN_TILES:KEY_H]
 [TOGGLE_OSD:KEYF_2]
 [TOGGLE_KEYBINDS:KEYS_SLASH]
+[TOGGLE_ANNOUNCEMENTS:KEY_A]
+[TOGGLE_DEBUG:KEYS_SEMICOLON]
```

Comment:
I think *this* should be the KEYS_TILDE to match other games. `gui/launcher` chose backtick/tilde as the hotkey because of it's association with "debug console"